EditableTable/components/InputComponents/DateTimeFormat.tsx

```diff
@@ -72,9 +74,15 @@ export const DateTimeFormat = React.memo(({ fieldName, dateOnly, value,
     timeKey = undefined;
   }
 
+  const checkValidation = () => {
+    if (isRequired && (currentDate === undefined || isNaN(currentDate.getTime()))) {
+      setInvalid(true);
+    }
+  };
+
   const onParseDateFromString = React.useCallback(
     (newValue: string): Date => parseDateFromString(_service, newValue),
-    [currentDate],
+    [],
   );
```

Comment:
Do we need this function at all? Can't we use the parseDateFromString method directly?